src/main/java/com/hubspot/jinjava/el/ext/ExtendedParser.java

```diff
@@ -104,6 +105,10 @@ public ExtendedParser(Builder context, String input) {
       CollectionMembershipOperator.TOKEN,
       CollectionMembershipOperator.HANDLER
     );
+    putExtensionHandler(
+      CollectionNonMembershipOperator.TOKEN,
+      CollectionNonMembershipOperator.getHandler(false)
```

Comment:
Suggest `CollectionNonMembershipOperator.HANDLER` instead to match the other putExtensionHandler calls